src/client/__init__.py

```diff
@@ -357,15 +382,12 @@ def _search_request(self, method, path, request, sort, limit):
         return self._request(method, path, params=params)
 
     def _find_all(self, find_function, request, sort_field: str = 'id', items_field: str = 'items'):
-        ctx = contextvars.copy_context()
-        yield
-
-        result = ctx.run(find_function, request, sort=[sort_field])
+        result = find_function(request, sort=[sort_field])
```

Comment:
Previously there was a hack to capture top-level contextvar. This is not necessary now thanks to `toloka.util._managing_headers.run_in_current_context`